exporter/consumer.go

```diff
@@ -186,20 +186,6 @@ func NewConsumer(config ConsumerConfig) (*Consumer, error) {
 	if err != nil {
 		return nil, errors.Annotate(err, "unable to create new kafka client")
 	}
-	topics, err := client.Topics()
```

Comment:
No longer necessary? What happens if the topic doesn't exist now? Does that come through c.group.Errors() (and have you confirmed this)?

Reply:
Yes, sarama will auto retry until the topic exists and outputs the error.